packages/stryker-elements/src/lib/molecules/toggle-repository.ts

```diff
@@ -25,60 +17,39 @@ export class ToggleRepository extends BaseElement {
   render() {
     return html`
       <div
-        class="${classMap({
-          'animate-pulse': this.loading,
-        })}grid grid-cols-2 rounded-lg border-2 border-neutral-600 p-3"
+        @click="${this.#handleClick}"
+        class="grid cursor-pointer grid-cols-2 rounded-lg border-2 border-neutral-600 p-2"
       >
-        <span class="font-bold text-white">${this.name}</span>
-        ${this.#apiKeyCopiedOrNoApiKeyAvailable()
-          ? this.#renderToggleButton()
-          : this.#renderApiKey()}
+        <span class="ms-2 flex items-center text-lg font-bold text-white">${this.name}</span>
```

Comment:
For enabled repositories, can you add the score badge to here as well? And have a link to the reports page from here (if the repository is enabled)

<img width="663" alt="image" src="https://github.com/user-attachments/assets/0fcc4b5b-d1fe-4364-99dd-835c9e404399">